tests/Rector/CodeQuality/SimplifyArraySearchRector/Wrong/wrong.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+
+if (array_search('searching', $array) !== false) {
+    echo 'yes!';
+}
+
+if (array_search('searching', $array) === true) {
```

Comment:
This refactoring is actually wrong: `array_search` returns false if didn't find the **index**, otherwise, the index itself. Also, it has a third argument `$strict`. I'll refactor the implementation, as well handle this argument

Reply:
Thanks